packages/sourcemap-upload/README.md

```diff
@@ -28,50 +28,68 @@ and vice-versa.
 This module exports a named `uploadSourceMaps` function with the following options:
 
 ```typescript
-interface Options {
+export interface UploadOptions {
   /**
    * The files/directories to search for sourcemaps. All files that match the
    * 'extensions' list and fail to match 'ignore' will be searched for
    * sourcemap JSON or `//#sourceMappingURL=` coments in order to find pairs
    * of generated-file + sourcemap, and the sourcemap will be uploaded.
    */
   filepaths: Array<string> | string;
-
   /**
    * To allow for tracking and browsing of maps that have been uploaded, we
    * require uploaded sourcemaps to have an overall group name associated with
    * them. This could for instance be a version number, or commit hash.
    */
   group: string;
-
   /**
    * The API key to use when connecting to Replay's servers.
    * Defaults to `process.env.REPLAY_API_KEY`.
    */
   key?: string;
-
   /**
    * Run all of the local processing and searching for maps, but skip uploading them.
    */
   dryRun?: boolean;
-
+  /**
+   * Delete all found sourcemap files after they have been uploaded.
+   */
+  deleteAfterUpload?: boolean;
```

Comment:
So we'll recommend people switch to this from `webpack-delete-sourcemaps-plugin` if they happen to use that plugin. The goal of this option is to serve as a replacement for that other thing, right?

Reply:
Yep!  Right now our doc page at https://docs.replay.io/reference/replay-cli/source-maps#next-js has a fictional `deleteSourcemaps()` function in the example. I was going to update that to direct people to use the deletion plugin, ran into issues, and said "I'll just do it myself".  Plus it's simpler this way anyway :)